cypress/e2e/authentification.cy.ts

```diff
@@ -7,23 +7,17 @@ describe("Authentification", () => {
         cy.wait("@graphqlRequest");
     });
 
-    it("Success authentification", () => {
+    xit("Success authentification", () => {
```

Comment:
Please put a comment why test is excluded.

Reply:
Left comment